src/components/ListCompanies/ListCompanies.jsx

```diff
@@ -0,0 +1,38 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Button } from 'react-bootstrap';
+import './styles.less';
+
+const propTypes = {
+    companies: PropTypes.arrayOf(PropTypes.shape({
+        id: PropTypes.string,
+        name: PropTypes.string,
+        phone: PropTypes.string,
+        revenue: PropTypes.string,
+    })),
+};
```

Comment:
PropTypes are really required, but when you do code challenges, people like to see explicit entries like this showing you know what the data will look like... this isn't always the case but it's always a good goal to keep in mind.